sdk/workloads/Azure.ResourceManager.Workloads/tests/SingleServerInstall.json

```diff
@@ -0,0 +1,62 @@
+{
+    "identity": null,
+    "properties.environment": "NonProd",
+    "properties.sapProduct": "S4HANA",
+    "properties": {
+        "configuration": {
```

Comment:
I didn't see the properties used as discriminator in this file. To make following code works, you should use the real response here.
```csharp
var deploymentConfiguration =
    DeploymentConfiguration.DeserializeDeploymentConfiguration(
        deploymentConfigurationJson.GetProperty("properties").GetProperty("configuration"));
```